src/ring_theory/ideal/operations.lean

```diff
@@ -859,6 +859,27 @@ end
 
 end ideal
 
+lemma ideal_mul_eq_zero (R : Type* )[integral_domain R] {I J : ideal R} : (I * J = ⊥) ↔ I = ⊥ ∨ J = ⊥ :=
```

Comment:
```suggestion
lemma ideal_mul_eq_bot (R : Type*) [integral_domain R] {I J : ideal R} : (I * J = ⊥) ↔ I = ⊥ ∨ J = ⊥ :=
```
I think it's standard in mathlib to refer to this ideal as `bot` rather than `zero`. Also a bit of spacing.